types/template.d.ts

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
```

Comment:
maybe rename it to mustache.d.ts as there is markdown.d.ts

Reply:
or else add it into additional.d.ts

Reply:
Done